src/components/LayoutTree.tsx

```diff
@@ -617,6 +722,27 @@ const LayoutTree = () => {
         marginTop: '0.501em',
       }}
     >
+      {hoverArrowVisibility && (
+        <div
+          style={{
+            width: '0',
+            height: '0',
+            borderLeft: '10px solid transparent',
+            borderRight: '10px solid transparent',
+            position: 'absolute',
+            top: hoverArrowVisibility === 'above' ? scrollTop : undefined,
+            left: '50%',
+            zIndex: '99999',
+            transform: 'translateX(-50%)',
+            borderBottom: '20px solid rgb(155, 170, 220)',
+            ...(hoverArrowVisibility === 'below' && {
+              bottom: `${arrowBottom}px`,
+              rotate: '180deg',
+            }),
+            animation: `bobble ${token('durations.arrowBobbleAnimation')} infinite`,
+          }}
+        ></div>
+      )}
```

Comment:
I recommend breaking this into a separate component with its own name such as `<HoverArrow>`. Being defined inline here without a name makes the intended functionality non-obvious to anyone skimming through the code.